src/camp/generators/create_sql.py

```diff
@@ -76,15 +76,15 @@ def __init__(self, admset, adm, out_path, dialect):
         self._var_prefix = "@" if self.dialect == 'mysql' else ''
         self._vars_def = set()
         self._vars_use = set()
+        self.sql_name = cu.yang_to_sql(self.adm.norm_name)
```

Comment:
@jenny-lee-cao There are a few references to `camputil` that need to get cleaned up in this module. But no need to hold up the PR since we know SQL changes still need to be made.